tests/vmipreset_test.go

```diff
@@ -380,8 +380,6 @@ var _ = Describe("[rfe_id:609][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Verifying VMI")
-			Expect(newVmi.Annotations).To(BeNil())
```

Comment:
isn't this one important ? (I don't know enough about presets to tell)
maybe this should be replaced by checking that we only have the testsuite's annotation?

Reply:
Now that there is a default annotation always added, I guess this assertion is not correct anymore.
But it makes sense to ask if the intention here was to check that no one else is adding a new annotation. Perhaps this assert should be changed with checking the original annotation from before the creation and the one after. (i.e. `Expect(vmi.Annotations).To(Equal(newVmi.Annotations))`)